addons/project/models/project_task.py

```diff
@@ -1077,6 +1077,22 @@ def write(self, vals):
         if "personal_stage_type_id" in vals and not vals['personal_stage_type_id']:
             del vals['personal_stage_type_id']
 
+        # sends an email to the 'Task Creation' subtype subscribers
+        # When project_id is changed
+        if vals.get('project_id'):
+            project = self.env['project.project'].browse(vals.get('project_id'))
+            notification_subtype_id = self.env.ref('project.mt_project_task_new').id
+            partner_ids = project.message_follower_ids.filtered(lambda follower: notification_subtype_id in follower.subtype_ids.ids).mapped('partner_id.id')
```

Comment:
Equivalent to:
```suggestion
            partner_ids = project.message_follower_ids.filtered(lambda follower: notification_subtype_id in follower.subtype_ids.ids).partner_id.ids
```

Reply:
Hello @xavierbol 
My bad :innocent: 

I've implemented the changes